src/components/Teams/TeamMembersPopup.jsx

```diff
@@ -16,6 +16,7 @@ export const TeamMembersPopup = React.memo(props => {
   const [selectedUser, onSelectUser] = useState(undefined);
   const [isValidUser, onValidation] = useState(true);
```

Comment:
Can these 2 state update functions (lines 16 and 17) names be updated to follow the convention? The convention for naming the state update functions is to use a prefix like "set" followed by the state variable name.